.github/workflows/rust.yml

```diff
@@ -26,14 +26,14 @@ jobs:
         run: cargo fmt -- --check
 
       - name: Build (release) (default features)
-        run: cargo build --target i686-pc-windows-msvc --locked --release
+        run: cargo build -Z build-std --target i686-win7-windows-msvc --locked --release
```

Comment:
We should probably add instructions to the readme regarding building for windows 7.

a la
https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/platform-support/win7-windows-msvc.md